Protos/api.proto

```diff
@@ -87,5 +87,10 @@ message TaskDataParam {
     oneof dataParam {
         Future future = 1;
         bytes data = 2;
+        FutureList futures = 3;
```

Comment:
Why is this needed and where it is used?

Reply:
used in addition operation